package.json

```diff
@@ -136,7 +137,8 @@
     "require-reload": "^0.2.2",
     "simple-git": "^3.19.1",
     "storybook": "7.0.20",
-    "styled-components": "5.3.9",
+    "styled-components": "^6.0.0",
+    "stylis": "^4.0.0",
```

Comment:
Will `stylis` be included when we install `styled-components 6`?

I think we can remove.

Reply:
I was thinking stylis was a PeerDependency of Styled Components but looks like it is just a regular dependency so yes I think we can remove it here.